config/initializers/check_libssl.rb

```diff
@@ -0,0 +1,14 @@
+#
+# This check is required because libssl1.1, which is deprecated, is required by
+# wkhtmltopdf-binary (0.12.6.6), which was released Nov 30, 2022.
+#
+# This file can be removed if wkhtmltopdf-binary 0.12.6.6 is no longer used.
+#
+
+unless File.exist?('/usr/lib/x86_64-linux-gnu/libssl.so.1.1')
```

Comment:
This is Linux-specific... there must be a way to search for native shared libs in Ruby that is independent of the system. Well, considering that CTRL is officially deployed on Linux and we might move away from Rails I'll accept this for now.